pkgs/by-name/ki/kikit/default.nix

```diff
@@ -35,9 +35,38 @@ buildPythonApplication rec {
     owner = "yaqwsx";
     repo = "KiKit";
     tag = "v${version}";
-    hash = "sha256-mwe/CajmQD5nDNYtQXRQm4vIJJiY7P5uFrvn8Ngjqc4=";
+    hash = "sha256-eemPRqfZ4Yer8AIEjx8UW9+ryCB9UEL9Zb7Z6gNN9Z4=";
+    # Upstream uses versioneer, which relies on gitattributes substitution.
+    # This leads to non-reproducible archives on GitHub.
+    # See https://github.com/NixOS/nixpkgs/issues/84312
+    postFetch =
+      let
+        patchedVersioneer =
+          # python
+          writeTextFile {
+            name = "versioneer.py";
+            text =
+              # python
+              ''
+                def get_version():
+                  return "${version}"
+
+                def get_cmdclass(*args):
+                  return {}
+              '';
+          };
+      in
+      ''
+        cp ${patchedVersioneer} $out/versioneer.py
+      '';
```

Comment:
This doesn't touch the hash-changing file.  While it might be stable from now on, we're likely to hit the same on future update as well.  Adding such a `versioneer.py` wouldn't even need to be done during fetch, but the deletion (or replacement) needs to be.

Reply:
Oops, you're totally right. I've just forced-pushed a v2 of things. PTAL.